gpii/node_modules/flowManager/src/BrowserChannel.js

```diff
@@ -40,8 +42,10 @@ gpii.flowManager.browserChannel.receiveMessage = function (that, message, soluti
     }
     solutionsRegistryDataSource.get({os: platformReporter.reportPlatform().id}, function onSuccess(entries) {
         if (!(solutionId in entries)) {
-            gpii.flowManager.browserChannel.sendError(that, "Rejecting a connection request from '" + solutionId +
-                      "'. The solution id was not found in the solutions registry");
+            gpii.flowManager.browserChannel.sendError(
```

Comment:
If this is a forced change, it is undesirable since we should have more freedom than this to wrap our lines

Reply:
I reverted this change.